### PR TITLE
ci: add extended support for windows-11

### PR DESCRIPTION
**Pull request by v1v**
As stated in https://www.elastic.co/support/matrix
 add support for Windows-11, for all the beats but `winlogbeat` and `functionbeat`

### CODE REVIEW

**Comment by andrewkroh** [comment]
Does this fail for Packetbeat with Windows 11? I'm surprised by the lack of Windows 10 and 11 in the matrix for Packetbeat. I don't recall if this was on purpose? @jamiehynds, do you know why Windows 10/11 is omitted for Packetbeat 8.x.

**Comment by v1v** [comment]
> Does this fail for Packetbeat with Windows 11? 

Yes, I tried a couple of times with the same outcome. See the [test-history](https://beats-ci.elastic.co/job/Beats/job/beats/job/PR-32236/lastCompletedBuild/testReport/history/)


image

**Comment by jamiehynds** [comment]
> Does this fail for Packetbeat with Windows 11? I'm surprised by the lack of Windows 10 and 11 in the matrix for Packetbeat. I don't recall if this was on purpose? @jamiehynds, do you know why Windows 10/11 is omitted for Packetbeat 8.x.

@andrewkroh I doubt it was on purpose, and most likely an error. I can get the Support Matrix updated ASAP.

**Comment by v1v** [comment]
`Packetbeat` is now using windows-11 as per https://github.com/elastic/beats/pull/32432

File: x-pack/winlogbeat/Jenkinsfile.yml [hunk 1]
```diff
@@ -49,6 +49,11 @@ stages:
         platforms:             ## override default labels in this specific stage.
             - "windows-2012-r2"
         stage: extended_win
+    windows-11:
+        mage: "mage build unitTest"
+        platforms:             ## override default labels in this specific stage.
+            - "windows-11"
+        stage: extended_win
```
Comment (v1v) [inline x-pack/winlogbeat/Jenkinsfile.yml:56]:
```suggestion
   # See https://github.com/elastic/beats/issues/32469
   # windows-11:
   #     mage: "mage build unitTest"
   #     platforms:             ## override default labels in this specific stage.
   #         - "windows-11"
   #     stage: extended_win
```

File: winlogbeat/Jenkinsfile.yml [hunk 2]
```diff
@@ -44,6 +44,11 @@ stages:
         platforms:             ## override default labels in this specific stage.
             - "windows-2012-r2"
         stage: extended_win
+    windows-11:
+        mage: "mage build unitTest"
+        platforms:             ## override default labels in this specific stage.
+            - "windows-11"
+        stage: extended_win
```
Comment (v1v) [inline winlogbeat/Jenkinsfile.yml:51]:
```suggestion
    # See https://github.com/elastic/beats/issues/32468
    #windows-11:
        #mage: "mage build unitTest"
        #platforms:             ## override default labels in this specific stage.
        #    - "windows-11"
        #stage: extended_win
```

**Review by cachedout — APPROVED** [review]

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**